integration/spark/src/test/scala/org/apache/spark/carbondata/TestStreamingTableWithLongString.scala

```diff
@@ -108,7 +109,16 @@ class TestStreamingTableWithLongString extends QueryTest with BeforeAndAfterAll
     )
 
     val row = sql("select * from streaming_longstr.stream_table_longstr_file order by id").head()
-    val exceptedRow = Row(10, "name_10", "city_10", 100000.0, BigDecimal.valueOf(0.01), 80.01, Date.valueOf("1990-01-01"), Timestamp.valueOf("2010-01-01 10:01:01.0"), Timestamp.valueOf("2010-01-01 10:01:01.0"), "10" + longStrValue)
+    val exceptedRow = Row(10,
```

Comment:
can enable scala style off/on to avoid changes in this file

Reply:
done